ansible_ai_connect/ai/api/telemetry/schema1.py

```diff
@@ -234,3 +234,4 @@ class ChatBotOperationalEvent(ChatBotBaseEvent):
 @define
 class StreamingChatBotOperationalEvent(ChatBotBaseEvent):
     event_name: str = "streamingChatOperationalEvent"
+    phase: str = "init"
```

Comment:
What's the need to set the `init` phase in Chatbot streaming event

Reply:
It is not needed, but it would make event filtering easier on Amplitude.